mne_bids/tests/test_write.py

```diff
@@ -1625,3 +1626,71 @@ def test_mark_bad_channels(_bids_validate,
     assert 'status_description' in tsv_data
     for description in expected_descriptions:
         assert description in tsv_data['status_description']
+
+
+def test_write_fine_calibration(_bids_validate):
```

Comment:
can you add a test what happens when a dict is passed? --> (1) a valid dict as read from the mne.preprocessing func, and (2) an arbitrary dict (which should raise an error)

Reply:
Addressed in eb6b3fe